features/guidance/turn-angles.feature

```diff
@@ -773,7 +773,7 @@ Feature: Simple Turns
 
         When I route I should get
             | waypoints | route                   | turns                               |
-            | a,j       | Siemens,Siemens,Siemens | depart,continue slight right,arrive |
+            | a,j       | Siemens,Siemens,Siemens | depart,continue straight,arrive     |
```

Comment:
this change looks weird at first, but if you consider two roads `(a--b)` and `(i--j)` with half a meter offset in between the two, going straight is actually a reasonable resposne here.